tools/create_index/index_generator.js

```diff
@@ -0,0 +1,37 @@
+var fs = require('fs')
+
+// We will fill in this object with our data
+var vuln = {}
+
+const createIndex = function (vulnDirectoryPath) {
+  const files = fs.readdirSync(vulnDirectoryPath)
+  getVulnDirectoryContents(files, vulnDirectoryPath)
+  writeIndex(vuln, vulnDirectoryPath)
+}
+
+const getVulnDirectoryContents = function (files, vulnDirectoryPath) {
+  for(file of files) {
+    const filename = file.slice(-0, file.toString().indexOf('.json'))
+    if(filename !== 'index') {
+      const data = fs.readFileSync(vulnDirectoryPath + file)
+      createVulnObject(filename, data)
+    }
+  }
+}
+
+const createVulnObject = function(identifier, json) {
+  vuln[identifier] = JSON.parse(json)
+}
+
+const writeIndex = function(data, vulnDirectoryPath) {
+  fs.writeFile(vulnDirectoryPath + 'index.json', JSON.stringify(data), function (error) {
```

Comment:
perhaps I'd just turn this one into a `writeFileSync` to be consistent with the synchronous style of the rest of the file